compiler-core/src/config.rs

```diff
@@ -733,6 +758,12 @@ impl Repository {
             Repository::BitBucket { repo, user } => {
                 Some(format!("https://bitbucket.com/{user}/{repo}"))
             }
+            Repository::CodeBerg { repo, user } => {
+                Some(format!("https://codeberg.org/{user}/{repo}"))
+            }
+            Repository::Forgejo { repo, user, host } | Repository::Gitea { repo, user, host } => {
+                Some(format!("https://{host}/{user}/{repo}"))
```

Comment:
Let's only add the `https://` if it or `http://` has not already been specified.